samples/outlook/preview-apis-signature/work-with-client-signatures.yaml

```
@@ -0,0 +1,115 @@
+id: outlook-signature-work-with-client-signatures
+name: Work with Client Signatures
+description: 'Check if the client signature is enabled, disable the client signature, get the compose type, and set a signature.'
+host: OUTLOOK
+api_set:
+    Mailbox: '1.1'
+script:
+    content: |
+        $("#isClientSignatureEnabled").click(isClientSignatureEnabled);
+        $("#disableClientSignature").click(disableClientSignature);
+        $("#getComposeType").click(getComposeType);
+        $("#setSignature").click(setSignature);
+
+        function isClientSignatureEnabled() {
+          // Check if the client signature is currently enabled
+          Office.context.mailbox.item.isClientSignatureEnabledAsync(function(asyncResult) {
+            if (asyncResult.status === Office.AsyncResultStatus.Succeeded) {
+              console.log("isClientSignatureEnabledAsync succeeded with result: " + asyncResult.value);
+            } else {
+              console.error(asyncResult.error);
+            }
+          });
+        }
+
+        function disableClientSignature() {
+          // Disable the client signature
+          Office.context.mailbox.item.disableClientSignatureAsync(function(asyncResult) {
+            if (asyncResult.status === Office.AsyncResultStatus.Succeeded) {
+              console.log("disableClientSignatureAsync succeeded");
+            } else {
+              console.error(asyncResult.error);
+            }
+          });
+        }
+
+        function getComposeType() {
+          // Get the compose type of the current item (compose only)
```

If the entire set of APIs only apply in compose mode, then you don't need to specify here.
```suggestion
          // Get the compose type of the current item.
```